src/redux/reducers/index.js

```diff
@@ -62,20 +63,20 @@ export const resourceTypesReducer = (state = preloadedResourceIdsGroupedByType,
   }
 };
 
-const preloadResourceTypeFilters = {};
+const preloadResourceTypeFilters = Object.keys(RESOURCE_TYPES)
+  .sort((a, b) => {
+    if (a.toLowerCase() > b.toLowerCase()) return 1;
```

Comment:
if `preloadResourceTypeFilters` is still a hash, sorting the keys doesn't seem do anything.  
(unless we are relying on the default order for `Object.keys` of the result?)